packages/eth2.0-state-transition/src/constants/constants.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @module constants
+ */
+
+export const DEPOSIT_CONTRACT_TREE_DEPTH = 2 ** 5; // 32
+export const GENESIS_SLOT = 0;
+export const GENESIS_EPOCH = 0;
+export const GENESIS_START_SHARD = 0;
+export const FAR_FUTURE_EPOCH = Infinity;
+export const ZERO_HASH = Buffer.alloc(32, 0);
+export const EMPTY_SIGNATURE = Buffer.alloc(96, 0);
+export const SECONDS_PER_DAY = 86400;
+
+// Signature domains
+export enum DomainType {
+  BEACON_PROPOSER = 0,
+  RANDAO = 1,
+  ATTESTATION = 2,
+  DEPOSIT = 3,
+  VOLUNTARY_EXIT = 4,
+  TRANSFER = 5,
+}
```

Comment:
I would maybe move  this file to utils

Reply:
Why? Seems like in a constants folder makes sense to me

Reply:
It would be useful for a lot packages that doesn't need state transition and state transition already has eth2.0-utils dependency. Not a big deal though

Reply:
Ah, you meant the utils package... I thought you mean the utils folder in state-transition. I'd like to do that maybe in another PR later. This ones getting quite chunky already